### PR TITLE
Support Linux based OS

### PR DESCRIPTION
**Pull request by junkycoder**
Solved issues:
- dependencies install
- plugins install

Tested on OSX.

### CODE REVIEW

File: src/pluginsLoader.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ class PluginLoader implements IPluginLoader {
     }
 
     escapeCmdPath(path) {
-        return path.replace(/\//g, pathPlatformDependent.sep);
```
Comment (Bobris) [inline src/pluginsLoader.ts]: This should be already ok. I fixed this in meantime probably.

**Comment by Bobris** [comment]
I will merge it later today. It is up to you if you want to revert or change that one commented line.

**Comment by Bobris** [comment]
BTW look at shell option in https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options it could probably simplify that.

**Comment by junkycoder** [comment]
You are right, see the new PR https://github.com/Bobris/bobril-build/pull/22

**Pull request merged**